packages/flutter/test/rendering/mock_canvas.dart

```diff
@@ -831,7 +835,11 @@ class _TestRecordingCanvasPatternMatcher extends _TestRecordingCanvasMatcher imp
       return false;
     } on String catch (s) {
       description.writeln(s);
-      description.write('The stack of the offending call was:\n${call.current.stackToString(indent: "  ")}\n');
+      try {
+        description.write('The stack of the offending call was:\n${call.current.stackToString(indent: "  ")}\n');
```

Comment:
`call.current` will throw if called after all calls have been evaluated

Reply:
Is there a way we can catch this without relying on catching a `TypeError`? Catching that would hide other errors we might introduce later...